src/shared/AppModal.jsx/AppModal.jsx

```diff
@@ -0,0 +1,36 @@
+import { AiOutlineCloseCircle } from 'react-icons/ai';
+
+import Modal from '@mui/material/Modal';
+
+import { AppModalProps } from './AppModal.props';
+import { BoxStyled, IconButtonStyled } from './Modal.styled';
+
+/**
+ * AppModal component for rendering a AppModal window.
+ *
+ * @component
+ * @param {function} handleClose - The callback function to handle modal close.
+ * @param {boolean} open - A boolean indicating whether the modal is open or closed.
+ * @param {React.ReactNode} children - The content to be displayed within the modal.
+ * @returns {JSX.Element}
+ */
+
+export const AppModal = ({ handleClose, children, open }) => {
+  return (
+    <Modal keepMounted open={open} onClose={handleClose}>
+      <BoxStyled>
+        <IconButtonStyled
```

Comment:
<img width="871" alt="Screenshot 2023-11-13 at 15 33 42" src="https://github.com/Autumn-fairytale/AFT-front/assets/95439176/869490d3-e3ab-4502-95ce-3d05d2bce77f">
Icon can be so large?

Reply:
Ok. I will change it. In the future, we can always change this button depending on our final needs. But for the reviewsForm in modal, my button looks normal.